app/views/partials/logo.ejs

```diff
@@ -1,23 +1,13 @@
 <%
 let parts = [];
 
-// If url contains corona, make the rest of it yellow
-if (baseUrl.includes('corona')) {
-  parts = baseUrl.split('corona').map(part => {
-    return !part ? 'corona' : `<span style="color: #F2CC20;" >${part}</span>`
-  })
-// Otherwise make tld including dot yellow
-} else {
-  parts = baseUrl.split('.');
-  parts[parts.length - 1] = `<span style="color: #f7bb00;">.${parts[parts.length - 1]}</span>`
-}
-
-const name = parts.join('')
+// If url contains corona, make the rest it white
+const name =  baseUrl.replace('corona', '<span style="color: #CAEAE9;;" >corona</span>')
```

Comment:
double semi-colon here, could you fix that?

Reply:
fixed